scripts/collection_from_items.py

```diff
@@ -102,6 +109,7 @@ def main(args: List[str] | None = None) -> None:
     arguments = parse_args(args)
     uri = arguments.uri
     collection_id = arguments.collection_id
+    linz_slug = arguments.linz_slug
```

Comment:
This is called only once. You can call it L181 rather than assigning it to a variable.

Reply:
Basically, we generally inline single-use variables unless there's a good reason for the extra variable. One such good reason is if the original name and the preferred name in the current context have drifted, but we don't want to change the interface (yet).

Reply:
Agreed - was an oversight to leave that in.